src/components/CreateAutomationModal.tsx

```diff
@@ -1,19 +1,51 @@
-import { useContext } from "react"
+import { type MouseEvent, useContext } from "react"
 import { ModalContext } from "~/state/ModalContext"
 
-const CreateAutomationModal = () => {
+enum EventName {
```

Comment:
if you wanna reuse the enum elsewhere you might want to move it into some shared types file